acoular/sdinput.py

```diff
@@ -49,6 +49,11 @@ def __init__(self, *args, **kwargs):
     #: Sampling frequency of the signal, changes with sinusdevices
     sample_freq = Property(desc='sampling frequency')
 
+    _sample_freq = Int(0)
```

Comment:
should be Float

Reply:
Resolved in https://github.com/acoular/acoular/pull/285/commits/ec664a66f6f7ac2200f26f2d35283bc9418d92e0 as proposed